lib/mock-component/mock-component.ts

```diff
@@ -38,11 +38,19 @@ export function MockComponent<TComponent>(component: Type<TComponent>): Type<TCo
       });
     }
 
-    /* tslint:disable:no-empty */
-    registerOnChange = (fn: (value: any) => void) => {};
-    registerOnTouched = (fn: (value: any) => void) => {};
+    /* tslint:disable:no-empty variable-name */
+    __simulateChange = (rating: number) => {};
+    __simulateTouch = () => {};
+
+    registerOnChange = (fn: (value: any) => void) => {
+      this.__simulateChange = fn;
+    }
+
+    registerOnTouched(fn: () => void): void {
+      this.__simulateTouch = fn;
+    }
     writeValue = (value: any) => {};
-    /* tslint:enable:no-empty */
+    /* tslint:enable:no-empty variable-name */
```

Comment:
Do you mind grouping the tslint violators together so that the disables are only around them?